src/libdeno.rs

```diff
@@ -19,18 +19,18 @@ pub struct isolate {
 ///
 /// If "alloc_ptr" is null, this type represents a borrowed slice.
 #[repr(C)]
-pub struct deno_buf {
+pub struct DenoBuf {
```

Comment:
Is `#[repr(C)]` actually doing anything here if the name doesn't match? I guess struct names get compiled away - and so don't need to match the libdeno side - but in that case shouldn't we also remove the `#[repr(C)]`? 

I think there's some benefit to using the same names as `deno.h` to represent the same things (despite being against Rust style) - it allows people to grep easily. (That said, the above `isolate` struct does not match `deno.h`'s `Deno`...)